test/unit/query/search.js

```diff
@@ -267,27 +267,72 @@ module.exports.tests.query = function(test, common) {
              'should': [
                {
                  'match': {
-                   'address.number': 123
+                   'address.number': {
+                     'query': 123,
+                     'boost': 1
+                   }
                  }
                },
                {
                  'match': {
-                   'address.street': 'main st'
+                   'address.street': {
+                     'query': 'main st',
+                     'boost': 3
+                   }
                  }
                },
                {
                  'match': {
-                   'address.zip': 10010
+                   'address.zip': {
+                     'query': 10010,
+                     'boost': 3
+                   }
                  }
                },
                {
                  'match': {
-                   'admin1_abbr': 'NY'
+                   'admin1_abbr': {
+                     'query': 'NY',
+                     'boost': 3
+                   }
                  }
                },
                {
                  'match': {
-                   'alpha3': 'USA'
+                   'alpha3': {
+                     'query': 'USA',
+                     'boost': 5
+                   }
+                 }
+               },
+               {
+                 match: {
+                   admin0: 'new york'
```

Comment:
admin0 uses the standard peliasAdmin analyzer https://github.com/pelias/schema/blob/master/mappings/partial/admin.json#L3

This will allow partial matching - in this case, ```new jersey``` and ```new mexico``` will be partial matches. If you want to use ```keyword``` analyzer - you get 0 or 1 match.. for example, if ```New York``` is how its stored in the index - you can only match it with ```New York``` and not ```new york``` (yes, its case sensitive)